mitosheet/mitosheet/mito_dash/v1/spreadsheet.py

```diff
@@ -163,7 +207,11 @@ def send(response):
                         },
                     }
                 )
-            
+
+            # TODO: document this - why we need to do this
```

Comment:
TODO: explain these, and why they need to change. Also the `num_messages` thing.